mlos_bench/mlos_bench/tests/storage/trial_config_test.py

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+#
+"""
+Unit tests for saving and retrieving additional parameters of pending trials.
+"""
+
+from mlos_bench.storage import Storage
+from mlos_bench.tunables import TunableGroups
+
```

Comment:
Tests for config hash uniqueness, repeatability, reusability?

Reply:
Storage API currently does not have any methods to retrieve the individual configs, and I am not sure I want to expose that functionality on the mlos_bench side... For unit tests, however, I can tap into SQLite directly and check if the data is there. Let's discuss

Reply:
I think retrieving configs is going to be important when we get to the point of visualizations and analysis, but we can probably add it later.